lib/web_ui/lib/src/engine/text_editing.dart

```diff
@@ -66,6 +66,29 @@ class EditingState {
         baseOffset = flutterEditingState['selectionBase'],
         extentOffset = flutterEditingState['selectionExtent'];
 
+  /// Creates an [EditingState] instance using values from the editing element
+  /// in the dom.
+  ///
+  /// This element can be a InputElement or a TextAreaElement depending on the
```

Comment:
nit: make "InputElement" and "TextAreaElement" linkable.

Reply:
done.